### ISSUE TITLE
etcdv3 client warn message: lease keepalive response queue is full; dropping response send

### ISSUE THREAD

**Issue by mingjunyang** [google/trillian#2249]
env 
```text
etcd server: 3.3.25

ct_server: 1.1.1
log_signer: 1.3.11
log_server: 1.3.11

server info : Linux ip-xx.xx.xx.xx.ap-mmmm-1.compute.internal 4.14.198-152.320.amzn2.x86_64 #1 SMP Wed Sep 23 23:57:28 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
```

when server start running a few moment, the console print warning message.
```text
log_signer_1_d621790f498d | {"level":"warn","ts":"2020-10-28T02:52:39.049Z","caller":"clientv3/lease.go:524","msg":"lease keepalive response queue is full; dropping response send","queue-size":16,"queue-capacity":16}
log_server_1_891d40c2df74 | {"level":"warn","ts":"2020-10-28T02:52:44.006Z","caller":"clientv3/lease.go:524","msg":"lease keepalive response queue is full; dropping response send","queue-size":16,"queue-capacity":16}
```

I don't know this warning should or not affect long-term operation.

I think it might be related to this issues https://github.com/etcd-io/etcd/issues/9911 . the man report *server memory highly load due to too many connections*.

**Comment by AlCutter** [comment]
Hi,

This is not an error we've come across before, but it sounds like it might be an issue with etcd/etcd client internals.

I found this blog post which seems to be written by someone who had a similar issue and looked into what was causing it, perhaps it may give some insight:
https://blog.csdn.net/puss0/article/details/96481856

**Comment by px3303** [comment]
@AlCutter 

Fixed by #2655
